WordPress/Classes/Stores/UserPersistentStore.swift

```diff
@@ -1,6 +1,6 @@
 class UserPersistentStore: UserPersistentRepository {
-    static let standard = UserPersistentStore(defaultsSuiteName: defaultsSuiteName)
-    private static let defaultsSuiteName = "temporary.suite.name" // TBD
+    static let standard = UserPersistentStore(defaultsSuiteName: defaultsSuiteName)!
+    private static let defaultsSuiteName = WPAppGroupName // TBD
```

Comment:
Super minor and doesn't need to be done in this PR, the `TBD` comment can be removed.

Reply:
I'll make sure to remove it in the next one